scryfall_scraper.py

```diff
@@ -3,7 +3,7 @@
 import time
 
 
-def get_scryfall(setUrl):
+def get_scryfall(setUrl='https://api.scryfall.com/cards/search?q=++e:xln'):
     #getUrl = 'https://api.scryfall.com/cards/search?q=++e:'
     #setUrl = getUrl + code.lower()
```

Comment:
We don't want to hard code a set here.
The commented out code looks fine and should do the trick, no?

Reply:
This is just a default setting to help with debug

Actual URL is set here: https://github.com/Cockatrice/Magic-Spoiler/blob/ed2ea3945a194333b79ab593c62ae384d1a1412c/main.py#L134

Reply:
👍